deluge/ui/gtkui/glade/preferences_dialog.ui

```diff
@@ -2924,7 +2924,7 @@ used sparingly.</property>
                                       <object class="GtkEntry" id="entry_outgoing_interface">
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
-                                        <property name="tooltip_text" translatable="yes">IP address for outgoing BitTorrent connections. Leave this empty if you want to use the default.</property>
+                                        <property name="tooltip_text" translatable="yes">Interface adapter name for outgoing BitTorrent connections. Leave this empty if you want to use the default.</property>
```

Comment:
`Interface adapter name for outgoing BitTorrent connections. (Leave  empty for default).`